store/errors.go

```diff
@@ -70,8 +70,27 @@ func (e *ErrDownload) Error() string {
 	return fmt.Sprintf("received an unexpected http response code (%v) when trying to download %s", e.Code, e.URL)
 }
 
+type ErrPasswordPolicy map[string]stringish
```

Comment:
These types are not following the standard pattern correctly. They should all be FooError instead of ErrFoo. The latter is applied for values, while the former is applied to types. Okay to not fix here since this was wrong before already, but let's try to fix it at some point in the future.